src/modules/UI/scenes/AddToken/AddToken.ui.js

```diff
@@ -69,7 +69,7 @@ class AddToken extends Component<Props, State> {
                 style={[styles.currencyName]}
                 value={this.state.currencyName}
                 onChangeText={this.onChangeName}
-                autoCapitalize={'words'}
+                autoCapitalize="words"
```

Comment:
`autoCapitalize={'words'}` is the proper syntax and style. Please change other use of `autoCapitalize="words"` to  `autoCapitalize={'words'}`